Add publish date

durden opened the pull request:
Add 'last updated' date to bottom of guides.

durden commented:
@prtkgpt originally requested this as the 'publish date'.  However, I thought about it and don't see why we shouldn't have the last updated date on all guides.  Seems like it could still be useful for guides 'in-review'  and 'draft'.

Note that the date shown is the date in UTC +00 of the last time the guide text was updated, **not** the metadata.  This seems to correlate with what a reader would expect to see; they don't care too much about the metadata.

durden commented:
Here's a screenshot of the placement of the last updated date:

<img width="1246" alt="screenshot 2016-06-15 11 00 47" src="https://cloud.githubusercontent.com/assets/58063/16074107/45f77c88-32e9-11e6-9f7d-fc9810ee2269.png">

Also, this feature has been deployed to the [staging site](http://pskb-stage.herokuapp.com) so you can test out a live example.

What do you think @prtkgpt ?

prtkgpt commented:
@durden this is perfect for now. In a later day, we can consider adding original date and time of the last edit. @edroman any suggestions on this

durden committed:
Refactor function to get commit data from github api

durden committed:
Cleanup display of love it CTA and updated date so they are not indented

- This makes it more clear visually that these are not part of the guide's
  text.

durden committed:
Change article.last_updated to be property for last updated date of article content, not metadata

durden committed:
Auto-upgrade last_updated attribute for new property use

durden committed:
Refactor fetching commits from API out of contributors function

- Fetching commits could be useful on its' own in the future.
